docs/config/README.md

```diff
@@ -235,8 +235,17 @@ This behavior is consistent across all keys, each key self documents itself with
 
 ## Environment only settings (not in config)
 
-#### Worm
-Enable this to turn on Write-Once-Read-Many. By default it is set to `off`. Set ``MINIO_WORM=on`` environment variable to enable WORM mode.
+#### Usage crawler
+Data usage crawler is enabled by default, following ENVs allow for more staggered delay in terms of usage calculation.
+
+Example: Add a delay factor for crawler to sleep between crawling. By default this value is 10 microseconds, allowed maximum is 1 second. Following example sets this value to 30 microseconds.
+```sh
+export MINIO_DISK_USAGE_CRAWL_DELAY=30
+minio server /data
+```
+
```

Comment:
```suggestion
Data usage crawler is enabled by default, following ENVs allow for more staggered delay in terms of usage calculation. The crawler can be completely disabled by setting `MINIO_DISK_USAGE_CRAWL_ENABLE=off`. This of course means that data usage will no longer be updated.

The crawler adapts to the system speed and completely pauses when the system is under load. It is possible to adjust the speed of the crawler and thereby the latency of updates being reflected. 

The delays between each operation of the crawl can be adjusted by the `MINIO_DISK_USAGE_CRAWL_DELAY` environment variable. By default the value is `10`. This means the crawler will sleep 10x the time each operation takes. This will in most setups make the crawler slow enough to not impact overall system performance.

Setting `MINIO_DISK_USAGE_CRAWL_DELAY` to a *lower* value will make the crawler faster and setting it to 0 will make the crawler run at full speed (not recommended). Setting it to a higher value will make the crawler slower, but consume less resources.

Example:
'''sh
export MINIO_DISK_USAGE_CRAWL_DELAY=30
minio server /data
'''

This will decrease the crawler speed by a factor of 3, reducing the system resource use, but increasing the latency of updates being reflected in the totals.

```

Reply:
NOTE, the suggesting thingie seems broken. I had to replace the example backticks for it to work :/

Reply:
Personally do not want to encourage disabling crawler. We may even drop that in future, so we should avoid explicitly documenting them. It was intentionallly left out.